certified-connectors/Meisterplan/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7710 @@
+          "type": "array",
+          "example": [
+            {
+              "bookingDate": "2019-01-31",
+              "hours": 5.5
+            }
+          ],
+          "items": {
+            "$ref": "#/definitions/Bookings"
+          },
+          "x-ms-summary": "Bookings"
+        },
+        "mode": {
+          "type": "string",
+          "example": "REPLACE",
+          "enum": [
+            "ADD",
+            "REPLACE"
+          ],
+          "x-ms-summary": "Mode"
+        }
+      },
+      "title": "ActualTimeWorkedCreateOrUpdateRequest"
+    },
+    "ActualTimeWorkedResponse": {
+      "type": "object",
+      "properties": {
+        "bookedEntity": {
+          "$ref": "#/definitions/ActualsEntityLinkResponse"
+        },
+        "bookings": {
+          "type": "array",
+          "description": "Booking date and hours of this actual time worked entry",
+          "items": {
+            "$ref": "#/definitions/ActualsBookingResponse"
+          },
+          "x-ms-summary": "Bookings"
+        }
+      },
+      "title": "ActualTimeWorkedResponse"
+    },
+    "ActualsBookingResponse": {
+      "type": "object",
+      "properties": {
+        "bookingDate": {
+          "type": "string",
+          "format": "date",
+          "example": "2019-01-31",
+          "description": "Booking date of this actual time worked entry",
+          "x-ms-summary": "Booking Date"
+        },
+        "hours": {
+          "type": "number",
+          "format": "double",
+          "example": 5.5,
+          "description": "Actual time worked in hours",
+          "x-ms-summary": "Hours"
+        }
+      },
+      "title": "ActualsBookingResponse"
+    },
+    "ActualsEntityLinkResponse": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "example": "0b9e0e29-8d87-4db8-baa8-1b1a49e354da",
+          "description": "The id of the entity to which this actual time worked entry is linked",
+          "x-ms-summary": "Id"
+        },
+        "type": {
+          "type": "string",
+          "description": "The entity type for this actual time worked entry",
+          "enum": [
+            "RESOURCE",
+            "ROLE"
+          ],
+          "x-ms-summary": "Type"
+        }
+      },
+      "title": "ActualsEntityLinkResponse"
+    },
+    "AllProjectsPriorityResponse": {
+      "type": "object",
+      "properties": {
+        "rank": {
+          "type": "integer",
+          "format": "int32",
+          "example": 1,
+          "description": "Rank depends on the date range and portfolio filter.",
+          "x-ms-summary": "Rank"
+        },
+        "rankCategory": {
+          "type": "string",
+          "example": "REGULAR",
+          "enum": [
+            "ABOVE_MUST_HAVE",
+            "BELOW_CUT_OFF",
+            "REGULAR"
+          ],
+          "x-ms-summary": "Rank Category"
+        }
+      },
+      "title": "AllProjectsPriorityResponse"
```

Comment:
Since these are technical terms of our product, we assume that clients from all locales should be familiar with them, and they don't need to be translated.